app/controllers/repos_controller.rb

```diff
@@ -92,33 +93,32 @@ def show_releases
           benchmark_runs << latest_benchmark_run
         end
 
-        columns = ChartBuilder.new(benchmark_runs).build_columns do |benchmark_run|
+        chart_builder = ChartBuilder.new(benchmark_runs, benchmark_result_type)
+        chart_builder.build_columns do |benchmark_run|
           environment = YAML.load(benchmark_run.environment)
 
-          if environment.is_a?(Hash)
-            temp = ""
-
-            environment.each do |key, value|
-              temp << "#{key}: #{value}<br>"
-            end
+          version = { version: benchmark_run.initiator.version }
 
-            environment = temp
+          # If there is more information about the environment, we add it to `version` 
+          if environment.is_a?(Hash)
```

Comment:
Could this end up with 2 different results? Or does `environment` always have an `environment` key?